src/vs/code/electron-browser/processExplorer/media/processExplorer.css

```diff
@@ -4,25 +4,29 @@
  *--------------------------------------------------------------------------------------------*/
 
 html {
-	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif;
 	font-size: 13px;
 }
 
-html:lang(zh-Hans) {
-	font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Noto Sans", "Microsoft YaHei", "PingFang SC", "Hiragino Sans GB", "Source Han Sans SC", "Source Han Sans CN", "Source Han Sans", sans-serif;
-}
+/* Font Families (with CJK support) */
```

Comment:
@RMacfarlane please review this change, it copies the CSS rule we define for the workbench to have the exact same font stack.